ERROR:root:name 'self' is not defined
Traceback (most recent call last):
  File "C:\Users\vikto\Documents\Old computer files\SFA\Scramble_for_Africa.py", line 826, in <module>
    main_loop.main_loop(global_manager)
  File "C:\Users\vikto\Documents\Old computer files\SFA\modules\main_loop.py", line 156, in main_loop
    current_button.on_click()#prioritize clicking buttons that appear above other buttons and don't press the ones
  File "C:\Users\vikto\Documents\Old computer files\SFA\modules\notifications.py", line 71, in on_click
    self.remove()
  File "C:\Users\vikto\Documents\Old computer files\SFA\modules\action_notifications.py", line 427, in remove
    trial_utility.complete_trial(previous_roll, self.global_manager)
  File "C:\Users\vikto\Documents\Old computer files\SFA\modules\trial_utility.py", line 93, in complete_trial
    self.global_manager.get('money_tracker').change(confiscated_money, 'trial compensation')
NameError: name 'self' is not defined
